examples/flask/app.py

```diff
@@ -0,0 +1,119 @@
+from flask import Flask, request
+from werkzeug.exceptions import HTTPException
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+pets = [
+    {
+        'name': 'Kitty',
+        'category': 'cat'
+    },
+    {
+        'name': 'Coco',
+        'category': 'dog'
+    },
+    {
+        'name': 'Flash',
+        'category': 'cat'
+    }
+]
+
+
+class PetModel(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(10))
+    category = db.Column(db.String(10))
+
+
+@app.before_first_request
+def init_database():
+    """Create the table and add some fake data."""
+    db.create_all()
+    for pet_data in pets:
+        pet = PetModel(**pet_data)
+        db.session.add(pet)
+    db.session.commit()
+
+
+@app.errorhandler(HTTPException)
+def handle_http_errors(error):
+    return {'message': error.name}, error.code
+
+
+def pet_schema(pet):
+    return {
+        'id': pet.id,
+        'name': pet.name,
+        'category': pet.category
+    }
+
+
+@app.get('/')
+def say_hello():
+    return {'message': 'Hello!'}
+
+
+@app.get('/pets/<int:pet_id>')
+def get_pet(pet_id):
+    pet = PetModel.query.get(pet_id)
+    if pet is None:
+        return {'message': 'Pet not found.'}, 404
+    return pet_schema(pet)
+
+
+@app.get('/pets')
+def get_pets():
+    pets = PetModel.query.all()
+    return {'pets': [pet_schema(pet) for pet in pets]}
+
+
+@app.post('/pets')
+def create_pet():
+    data = request.json
+    if 'name' not in data or 'category' not in data:
+        return {'message': 'The name and category field is required.'}, 400
+    if len(data['name']) > 10:
+        return {'message': 'The length of the name must be between 0 and 10.'}, 400
+    if data['category'] not in ['cat', 'dog']:
+        return {'message': 'The category must be one of: dog, cat.'}, 400
```

Comment:
- The categories are hardcoded.
- There is a duplication of the categories, one time in the list, one time in text.

Why not make a top-level list or set, so it could be overwritten?